.run/Executor.run.xml

```diff
@@ -8,7 +8,7 @@
       <env name="ARMADA_EXECUTORAPICONNECTION_FORCENOTLS" value="true" />
       <env name="ARMADA_HTTPPORT" value="8085" />
       <env name="HOME" value="$USER_HOME$/" />
-      <env name="KUBECONFIG" value="$PROJECT_DIR$/.kube/internal/config" />
+      <env name="KUBECONFIG" value="$PROJECT_DIR$/.kube/external/config" />
```

Comment:
This is intentional.  The run config for the executor is incorrect as `internal` will attempt to connect via the kind network, which won't work from Goland.  `external` will attempt to connect on loclahost which is correct.